synapse/storage/databases/main/events_worker.py

```diff
@@ -339,6 +342,14 @@ def get_chain_id_txn(txn: Cursor) -> int:
             writers=["master"],
         )
 
+        self.db_pool.updates.register_background_index_update(
```

Comment:
Add some context

(comment hasn't been wrapped)

```suggestion
        # Added to accommodate some queries for the admin API in order to fetch/filter membership events by when it was received
        self.db_pool.updates.register_background_index_update(
```